src/connectors/snowflake/trulens/connectors/snowflake/connector.py

```diff
@@ -233,6 +213,49 @@ def _init_with_snowpark_session(
 
         print(f"Set TruLens workspace version tag: {res}")
 
+    def _set_up_database_args(
+        self,
+        database_args: Dict[str, Any],
+        snowpark_session: Session,
+        connection_parameters: Dict[str, str],
+        database_redact_keys: bool,
+        database_prefix: Optional[str],
+    ) -> Dict[str, Any]:
+        database_args = database_args or {}
+        # Set engine_params.
+        default_engine_params = {
+            "creator": lambda: snowpark_session.connection,
+            "paramstyle": "qmark",
+            # The following parameters ensure the pool does not allocate new
+            # connections that it will close. This is a problem because the
+            # "creator" does not create new connections, it only passes around
+            # the single one it has.
+            "max_overflow": 0,
```

Comment:
does `max_overflow` matter if `creator` reuses the same connection? if so, shouldn't `pool_size` also be configured?

Reply:
talked offline.